cmd/neofs-cli/modules/container/set_eacl.go

```diff
@@ -41,7 +39,7 @@ Container ID in EACL table will be substituted with ID from the CLI.`,
 		pk := key.GetOrGenerate(cmd)
 		cli := internalclient.GetSDKClientByFlag(ctx, cmd, commonflags.RPC)
 		force, _ := cmd.Flags().GetBool(commonflags.ForceFlag)
-		if !(force || flagVarsSetEACL.noPreCheck) {
+		if !(force) {
```

Comment:
```suggestion
		if !force {
```
surprised that all linters don't care btw